app/src/main/java/io/appium/uiautomator2/core/InteractionController.java

```diff
@@ -62,48 +67,92 @@ public boolean injectEventSync(final InputEvent event) throws UiAutomator2Except
         return injectEventSync(event, true);
     }
 
+    public boolean shouldTrackScrollEvents() {
+        Session session = AppiumUIA2Driver.getInstance().getSessionOrThrow();
+        Boolean trackScrollEvents = true;
+        if (session.hasCapability(TRACK_SCROLL_EVENT_CAP)) {
+          try {
+              trackScrollEvents = (Boolean) session.getCapability(TRACK_SCROLL_EVENT_CAP);
+              Logger.error("Capability '" + TRACK_SCROLL_EVENT_CAP + "': " + trackScrollEvents);
```

Comment:
is this meant to be an error log?

Reply:
Ah, no. That was just all that would show up in my logs without fiddling. I'll fix.